src/go/k8s/helm-chart/charts/redpanda-operator/README.md

```diff
@@ -0,0 +1,53 @@
+# Redpanda Operator
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.1.0](https://img.shields.io/badge/AppVersion-0.1.0-informational?style=flat-square)
+
+## Installation
+
+### Prerequisite
+
+Due to problems with helm updating the CRD and deletion can destroy all custom resource, the CRD
```

Comment:
My suggestion:

===
1. Install Redpanda operator CRDs:

 ```sh
 kubectl apply -k https://github.com/vectorizedio/redpanda/src/go/k8s/config/crd
 ```

> The first step is needed to install the CRD. This is due to problems with Helm updating the CRD.


2. Install the Redpanda operator:

 ```sh
 helm install --namespace redpanda-system --create-namespace redpanda-operator ./redpanda-operator
 ```

===

I'm not sure what you mean by "and deletion can destroy all custom resource.". If it needs to be there, could you rephrase/add?

Reply:
Ok. I don't have strong opinion on how the structure of the README.md should look like.

Reply:
In which directory do you run the `helm install --namespace redpanda-system --create-namespace redpanda-operator ./redpanda-operator` command? Ideally, this should be a helm repo.

Reply:
I wrote in the https://github.com/vectorizedio/redpanda/pull/814/files#diff-e74e37824e22e5f56d24747cd67c3f3d3a35ed5343a1a4b6ea09e6a2f2459821R34

Reply:
@rkruze I was thinking that helm chart work will be splitted accordingly:

- create helm chart
- create CI
- create helm repo based on github pages
- update README